snmp/tests/compose/data/netgear-switch.snmprec

```diff
@@ -0,0 +1,1499 @@
+1.3.6.1.4.1.4526.10.1.1.1.5.0|4|Jaded
+1.3.6.1.4.1.4526.10.1.1.1.6.0|4x|64726976696e67206b657074204a6164656420627574
+1.3.6.1.4.1.4526.10.1.1.1.7.0|4x|61637465642062757420627574206163746564206f78656e20746865697220616374656420666f7277617264
+1.3.6.1.4.1.4526.10.1.1.1.8.0|4x|7a6f6d6269657320666f727761726420627574204a61646564204a61646564206b65707420627574206f78656e
+1.3.6.1.4.1.4526.10.1.1.1.9.0|4x|111111111111
+1.3.6.1.4.1.4526.10.1.1.1.10.0|4x|616374656420717561696e746c79206f78656e207468656972
+1.3.6.1.4.1.4526.10.1.1.1.11.0|4x|717561696e746c79206f78656e
+1.3.6.1.4.1.4526.10.1.1.1.12.0|4x|7a6f6d62696573207468656972
+1.3.6.1.4.1.4526.10.1.1.1.13.0|4x|64726976696e67207a6f6d62696573207a6f6d6269657320717561696e746c7920717561696e746c79
+1.3.6.1.4.1.4526.10.1.1.1.14.0|4x|7468656972206f78656e206275742064726976696e67206f78656e207468656972
+1.3.6.1.4.1.4526.10.1.1.2.1.0|2|22
+1.3.6.1.4.1.4526.10.1.1.2.3.0|2|6
+1.3.6.1.4.1.4526.10.1.1.2.4.1.1.0|2|0
+1.3.6.1.4.1.4526.10.1.1.2.4.1.1.28|2|28
+1.3.6.1.4.1.4526.10.1.1.2.4.1.2.0|4x|74686569722064726976696e67
+1.3.6.1.4.1.4526.10.1.1.2.4.1.2.28|4x|6f78656e206f78656e204a61646564207468656972206b657074204a61646564206f78656e207a6f6d6269657320627574
+1.3.6.1.4.1.4526.10.1.1.2.4.1.3.0|4x|7a6f6d62696573207468656972206163746564207a6f6d626965732064726976696e67206163746564207468656972206f78656e
+1.3.6.1.4.1.4526.10.1.1.2.4.1.3.28|4x|746865697220717561696e746c79206f78656e207468656972207a6f6d62696573
+1.3.6.1.4.1.4526.10.1.1.3.1.1.10|2|10
+1.3.6.1.4.1.4526.10.1.1.3.1.1.28|2|28
+1.3.6.1.4.1.4526.10.1.1.3.1.2.10|4x|4a61646564206b657074206f78656e206163746564206f78656e20717561696e746c79
+1.3.6.1.4.1.4526.10.1.1.3.1.2.28|4x|64726976696e672064726976696e6720666f727761726420717561696e746c7920627574207a6f6d62696573207a6f6d6269657320627574
+1.3.6.1.4.1.4526.10.1.1.3.1.3.10|4x|64726976696e672062757420717561696e746c7920717561696e746c79206f78656e207a6f6d62696573206163746564
+1.3.6.1.4.1.4526.10.1.1.3.1.3.28|4x|6f78656e206b657074206163746564206b657074206163746564
+1.3.6.1.4.1.4526.10.1.1.4.1.0|2|23
+1.3.6.1.4.1.4526.10.1.1.4.2.0|2|11
+1.3.6.1.4.1.4526.10.1.1.4.3.0|66|94
+1.3.6.1.4.1.4526.10.1.1.4.4.0|66|42451
+1.3.6.1.4.1.4526.10.1.1.4.5.0|66|22
+1.3.6.1.4.1.4526.10.1.1.4.6.0|66|28313
+1.3.6.1.4.1.4526.10.1.1.4.7.0|66|10922
+1.3.6.1.4.1.4526.10.1.1.4.8.1.1.11|2|11
+1.3.6.1.4.1.4526.10.1.1.4.8.1.1.22|2|22
+1.3.6.1.4.1.4526.10.1.1.4.8.1.2.11|4x|64726976696e67206163746564204a6164656420627574
+1.3.6.1.4.1.4526.10.1.1.4.8.1.2.22|4x|666f7277617264206f78656e207a6f6d6269657320666f7277617264204a61646564206163746564
+1.3.6.1.4.1.4526.10.1.1.4.8.1.3.11|4x|717561696e746c79206f78656e2064726976696e6720717561696e746c79206b657074
+1.3.6.1.4.1.4526.10.1.1.4.8.1.3.22|4|Jaded
+1.3.6.1.4.1.4526.10.1.1.4.8.1.4.11|4x|6163746564206b65707420627574206b657074206f78656e207a6f6d6269657320717561696e746c79
+1.3.6.1.4.1.4526.10.1.1.4.8.1.4.22|4x|666f7277617264206275742064726976696e67
+1.3.6.1.4.1.4526.10.1.1.4.9.0|4x|4a61646564206163746564204a61646564
```

Comment:
the test is failing because here the snmprec is defined this metric as string when it's integer
you need to change the type and the value 
```suggestion
1.3.6.1.4.1.4526.10.1.1.4.9.0|2|4356
```